ChartIQDemo/ChartIQDemo/SearchStudiesViewController.swift

```diff
@@ -56,7 +56,7 @@ class SearchStudiesViewController: UITableViewController {
         super.viewDidLoad()
         
         searchTextField.attributedPlaceholder =
-            NSAttributedString(string: "Search for studies", attributes: [NSForegroundColorAttributeName : UIColor.white])
+            NSAttributedString(string: "Search for studies", attributes: convertToOptionalNSAttributedStringKeyDictionary([convertFromNSAttributedStringKey(NSAttributedString.Key.foregroundColor) : UIColor.white]))
```

Comment:
I have a feeling this could be written better. I would test it myself, but I am having trouble with my environment. I probably should not have ran `git clean -ffdx`. Heh.

Based on [this stackoverflow answer](https://stackoverflow.com/a/46459315), I think the following will work.

```
searchTextField.attributedPlaceholder = NSAttributedString(string: "Search for studies", attributes: [NSAttributedString.Key.foregroundColor: UIColor.white] as! [NSAttributedStringKey: Any])
```

Reply:
Ah, got the build to work finally. The follow seems to compile fine.

```
searchTextField.attribtuedPlaceholder = NSAttributedString(string: "Search for studies", attributes: [NSAttributedString.Key.foregroundColor: UIColor.white])
```